### ISSUE TITLE
Default value provided not working in type ahead search card when used in messaging extension

### ISSUE THREAD

**Issue by shubham-betterworks** [OfficeDev/Microsoft-Teams-Samples#455]
### Steps to reproduce:
1. Clone https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-type-ahead-search-adaptive-cards/nodejs.
2. Add messaging extension command in manifest.json
```json
"composeExtensions": [
    {
        "botId": "{{Microsoft-App-Id}}",
        "commands": [
            {
                "id": "dynamicsearch",
                "type": "action",
                "title": "Dynamic Search",
                "description": "Dynamic Search",
                "initialRun": false,
                "fetchTask": true,
                "context": [
                    "compose"
                ]
            }
        ]
    }
  ],
```
3. Add the following messaging extension invoke handler in TeamsBot
```js
handleTeamsMessagingExtensionFetchTask(context, action) {
    const card = CardFactory.adaptiveCard(this.adaptiveCardForDyanamicSearch());

    return {
      task: {
        type: "continue",
        value: {
          card,
        },
      },
    };
  }
```
4. In `onInvokeActivity` method change last line `return null` with `return super.onInvokeActivity(context);`
5. Add `"value": "static_option_1",` in `Input.ChoiceSet` component of [adaptiveCardForDyanamicSearch](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-type-ahead-search-adaptive-cards/nodejs/bots/teamsBot.js#L254)
6. Start up the bot service

### Actual Behaviour
When you open the messaging extension task module the input component renders `Static Option 1` as a placeholder
<img width="642" alt="Screenshot 2022-08-30 at 12 51 09 PM" src="https://user-images.githubusercontent.com/80667102/187382719-38e11f32-a271-453e-8b37-aa46a8916daa.png">

### Expected behaviour
When you open the messaging extension task module the input component should have `Static Option 1` as the default value.

This above default value works fine when you send this same adaptive card in chat instead of task module and input component is rendered with the value `Static Option 1` 
<img width="553" alt="Screenshot 2022-08-30 at 12 51 18 PM" src="https://user-images.githubusercontent.com/80667102/187382976-45c86796-427b-4a93-a28a-d2a595bb57a2.png">

**Comment by Meghana-MSFT** [comment]
We are looking into this issue, we will get back to you.

**Comment by Meghana-MSFT** [comment]
We tried this at our end, and we see the following behavior -

Adaptive card in chat - Works fine
![image](https://user-images.githubusercontent.com/81138249/187413884-6c03e5a3-b154-47de-bc95-85a3e3b0b1ac.png)

Messaging extension - It says 1 item selected but when expanded the item is not selected
![image](https://user-images.githubusercontent.com/81138249/187413816-7cd2f773-5303-4f60-9d8c-bf3a21d17c3b.png)

Could you confirm if you are seeing it as 1 item selected or a placeholder value within messaging extension.

**Comment by shubham-betterworks** [comment]
Yes in case we set `isMultiSelect` to true we get a placeholder stating `1 item(s) selected` but the value is not selected and checkbox is untick

**Comment by Meghana-MSFT** [comment]
Thank you for the confirmation, we will check internally and get back to you.

**Comment by shubham-betterworks** [comment]
@Meghana-MSFT any updates?

**Comment by Meghana-MSFT** [comment]
@shubham-betterworks - Apologies for the delay in response, A bug has been raised for the same. We will keep you posted on the updates.

**Comment by Meghana-MSFT** [comment]
Could you please share the below details as the engineering teams needs this information -

How did the customer initiate the ME flow? From engineering team's investigation, as long as the bot is added to a team or chat, and initiated the expected way, the ME works.

**Comment by shubham-betterworks** [comment]
Add the bot --> Go to the chat window with the bot --> Initiate the messaging extension command --> Search and select multiple options

ME pop-up opens but the issue here is that the default value provided is treated as a placeholder and not prefilled value for input

**Comment by ChetanSharma-msft** [comment]
Hello @shubham-betterworks - Sorry for delay in response.
It would be better if you can share a short video for your issue.

**Comment by shubham-betterworks** [comment]
@ChetanSharma-msft will share it ASAP

**Comment by Meghana-MSFT** [comment]
@shubham-betterworks - For the issue with default choices not being showed, engineering team is investigating. We will keep you posted.

**Comment by shubham-betterworks** [comment]
@ChetanSharma-msft @Meghana-MSFT 
When a default value is provided in the input.choiceset field with dynamic search functionality enabled in the adaptive card the messaging extension treats it as a placeholder and not an actual input value whereas the same card works fine when sent as a message from a bot in the chat


https://user-images.githubusercontent.com/80667102/196605214-394ca96a-abeb-4c69-bcad-d332e90d7716.mov

**Comment by Meghana-MSFT** [comment]
Thank you for the video, we have shared the videos with engineering team. we will get back to you.

**Comment by shubham-betterworks** [comment]
@Meghana-MSFT any updates ???

**Comment by ChetanSharma-msft** [comment]
Hello @shubham-betterworks - We are checking the updates with engineering team and get back to you once we get any updates or confirmation.

**Comment by Meghana-MSFT** [comment]
Update : Engineering team has confirmed that they are actively working on the issue mentioned. We currently do not have an ETA to share.

**Comment by shubham-betterworks** [comment]
any updates on this ?

**Comment by Meghana-MSFT** [comment]
@shubham-betterworks - We will check this and get back to you.

**Comment by Meghana-MSFT** [comment]
Hi @shubham-betterworks - Apologies for the delay, we are not able to repro this issue at our end. We followed the steps mentioned in the initial question. We checked in Android OS 14 and Teams version 1416/1.0.0.2025021303/0119

https://github.com/user-attachments/assets/3f166886-24b1-4bf5-a376-f388fe7bf831
